compiler/src/iree/compiler/Dialect/Flow/IR/FlowOps.td

```diff
@@ -1238,16 +1238,20 @@ def FLOW_TensorTraceOp : FLOW_Op<"tensor.trace", []> {
 // Parameterization Ops
 //===---------------------------------------------------------------------===//
 
-def Flow_DispatchDefaultWorkgroupCountOp :
-    FLOW_PureOp<"dispatch.default_workgroup_count"> {
-  let summary = [{represents computation that returns workgroup count}];
+def Flow_DispatchWorkgroupCountFromDagRootOp :
+    FLOW_PureOp<"dispatch.workgroup_count_from_dag_root"> {
+  let summary = [{
+    workgroup count computed based on iteration range of the root of the DAG
+    for ops within dispatch
+  }];
   let description = [{
-    During dispatch region formation, the `workload` captured is used
-    by the backends to compute the number of workgroups. This operation
-    is a placeholder that represents the computation that takes the `workload`
-    as operands and returns the workgroup count as results.
-    The backends are expected to lower this operation into the actual computation
-    that returns the number of workgroups.
+    By default the IREEs backend relies on tile + distribution of the root
+    of the DAG of ops within the dispatch to split the work amongst workgroups.
```

Comment:
nit: I'd also define this acronym (Directed Acyclic Graph, I assume?) the first time it's used in the description

Although, I haven't seen DAGs mentioned much in the context of MLIR (I hear terms like "block", "region", "use-def chain", "SSA value" more). Would some other terminology also work here?

Reply:
This is a DAG of ops. Ill expand it in the description.